Modules/@babylonjs/react-native-basekit/scripts/install.js

```diff
@@ -0,0 +1,174 @@
+const https = require('https');
+const zlib = require('zlib');
+const tar = require('tar');
+const path = require('path');
+const fs = require('fs');
+
+function getCachePathCandidates() {
+  return [
+    process.env.npm_config_sass_binary_cache,
+    process.env.npm_config_cache,
+  ].filter(function(_) { return _; });
+}
+
+function getCached(package, binaryFilename) {
+  return [getCachePathCandidates(), 'BabylonReactNative', package.version, binaryFilename].join('/');
+}
+
+function useCachedFile(filePath) {
+  if (fs.existsSync(filePath)) {
+    console.log(`Using cached file ${filePath}`);
+    const gunzip = zlib.createGunzip();
+    const untar = tar.extract({ cwd: process.cwd() }); // Extract to the current working directory
+
+    const sourceStream = fs.createReadStream(filePath); 
+    sourceStream
+    .pipe(gunzip) // Decompress the .tar.gz file
+    .pipe(untar) // Extract the tar contents to the cwd
+    .on('error', (err) => {
+      console.error('Error extracting .tar.gz file:', err);
+      process.exit(1);
+    })
+    .on('end', () => {
+      console.log('Extraction completed successfully.');
+    });
+    return true;
+  }
+  return false;
+}
+
+function getArgument(name) {
+  const flags = process.argv.slice(2), index = flags.lastIndexOf(name);
+
+  if (index === -1 || index + 1 >= flags.length) {
+    return null;
+  }
+
+  return flags[index + 1];
+}
+
+function getBinaryUrl(package, binaryFilename) {
+  var site = getArgument('--brn-binary-site') ||
+              process.env.BRN_BINARY_SITE  ||
+              process.env.npm_config_brn_binary_site ||
+              (package.nodeBabylonConfig && package.nodeBabylonConfig.binarySite) ||
+              'https://github.com/CedricGuillemet/BabylonReactNative-1/releases/download';
```

Comment:
should this stay on github? and if yes - should it stay on your github (and not babylon)?

Reply:
no, it will be replaced with Babylon github